plugin.php

```diff
@@ -34,12 +34,22 @@
 
 class AgreableVenuesPlugin
 {
+  const FILE_LOCATION = "/tmp/kitchin-brand-response.json";
+
 	public function __construct()
 	{
     $ns = Helper::get('agreable_namespace');
     add_filter("acf/load_field/key={$ns}_plugin_brand", array($this, 'loadBrands'), 11, 3);
+    add_action('acf/save_post', array($this, 'clearCachedResponse'), 20);
   }
 
+  public function clearCachedResponse() {
+    $screen = get_current_screen();
+    if (strpos($screen->id, "acf-options") == true) {
```

Comment:
Will `$screen` *always* come back with an object that has the property`id`? If this the save_post was executed on the CLI then maybe there'd be no `$screen`? Potench